sormas-api/src/main/java/de/symeda/sormas/api/dashboard/DashboardContactDto.java

```diff
@@ -23,18 +21,22 @@
 import java.util.Map;
 
 import de.symeda.sormas.api.Disease;
+import de.symeda.sormas.api.audit.AuditInclude;
+import de.symeda.sormas.api.audit.AuditedClass;
 import de.symeda.sormas.api.contact.ContactClassification;
 import de.symeda.sormas.api.contact.ContactStatus;
 import de.symeda.sormas.api.contact.FollowUpStatus;
 import de.symeda.sormas.api.visit.VisitStatus;
 
+@AuditedClass
```

Comment:
This would lead to a lot of log data that doesn't really bring benefit. Focus on logging the DashboardCriteria and other parameters used for doing the queries instead.